book/src/installation-binaries.md

```diff
@@ -36,3 +36,17 @@ Each binary is contained in a `.tar.gz` archive. For this example, lets use the
 1. Test the binary with `./lighthouse --version` (it should print the version).
 1. (Optional) Move the `lighthouse` binary to a location in your `PATH`, so the `lighthouse` command can be called from anywhere.
     - E.g., `cp lighthouse /usr/bin`
+
+## Troubleshooting
+
+If you get a SIGILL (exit code 132), then your CPU is incompatible with the optimized build
```

Comment:
I guess this concept also applies to "Build from Source" as well. I notice it doesn't have any info about portability.

I'll leave it up to you to decide if you want to address that here.

Reply:
It shouldn't apply to source builds unless the user forces ADX on by bypassing the makefile, or moves a binary from an ADX machine to a non-ADX one. For now I'm in favour of merging ASAP, and we can add a note later if people run into it